source/_integrations/schedule.markdown

```diff
@@ -135,48 +122,53 @@ schedule:
           default: {}
 {% endconfiguration %}
 
-### Attributes
+## Attributes
 
-A schedule entity's state exports attributes that can be useful in
-automations and templates.
+A schedule entity's state exports attributes that can be useful in automations and templates.
 
 | Attribute | Description |
 | ----- | ----- |
 | `next_event` | A datetime object containing the next time the schedule is going to change state. |
+| Additional data | The entries in **Additional data** / `data` of a schedule block when the respective block is active. |
```

Comment:
Use the attribute key here, then use the description to re-mention that we're talking about addtional data.

Reply:
I concur here. Is this better?